ElectronClient/theme.js

```diff
@@ -417,19 +237,24 @@ function themeStyle(theme) {
 
 	// All theme are based on the light style, and just override the
 	// relevant properties
-	output = Object.assign({}, globalStyle, fontSizes, lightStyle);
+	output = Object.assign({}, globalStyle, fontSizes, themes[Setting.THEME_LIGHT]);
 
 	switch (theme) {
 	case Setting.THEME_DARK :
-		output = Object.assign({}, output, darkStyle); break;
+		output = Object.assign({}, output, themes[Setting.THEME_DARK]); 
+		break;
 	case Setting.THEME_SOLARIZED_LIGHT :
-		output = Object.assign({}, output, solarizedLightStyle); break;
+		output = Object.assign({}, output, themes[Setting.THEME_SOLARIZED_LIGHT]); 
+		break;
 	case Setting.THEME_SOLARIZED_DARK :
-		output = Object.assign({}, output, solarizedDarkStyle); break;
+		output = Object.assign({}, output, themes[Setting.THEME_SOLARIZED_DARK]); 
+		break;
 	case Setting.THEME_DRACULA :
-		output = Object.assign({}, output, draculaStyle); break;
+		output = Object.assign({}, output, themes[Setting.THEME_DRACULA]); 
+		break;
 	case Setting.THEME_NORD :
-		output = Object.assign({}, output, nordStyle); break;
+		output = Object.assign({}, output, themes[Setting.THEME_NORD]); 
+		break;
```

Comment:
Thanks for the update! Now that you have the themes in an object, I believe all this code can be simplified down to `output = Object.assign({}, globalStyle, fontSizes, themes[Setting.THEME_LIGHT], themes[theme]);` So there's no need for the switch case anymore, but please confirm by running the app and switching themes.

Reply:
Right, I'll give it a go! Once this pull-request get's approved (if I manage too. :)) I'll make an updated to my new theme and add it through this way. So I will remove that pull-request soon as it will need to be re-written to some small degree.

Reply:
Yep, that worked without any problems! So I have just submitted the updated code for it!

Reply:
I've also closed my other pull-request now and will re-submit once this is added. :) Since this is using a new structure and I would need to update my theme anyway once this goes in.